src/state/assets/selectors.js

```diff
@@ -1,6 +1,12 @@
 import { createSelector } from 'reselect';
 
 export const getAssetsState = state => state.apps.cms.assets;
+export const getApiState = state => state.api;
+
+export const getApiUrl = createSelector(
+  getApiState,
+  api => api.domain,
+);
```

Comment:
I suggest you omit this and use the existing method from @entando/apimanager's `getDomain`. It does exactly the same on what you're intend to do.